pkgs/markdown/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 7.3.2
+
+* Fix an issue with nested list structure when indented by tabs (#2172)
```

Comment:
Sorry, could you add this to the `7.3.1-wip` section, please? 7.3.1 is not released yet.